code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,22 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons)) /////addendum by flimflamm
+			var/decalselection = input("Please select a decal") as null|anything in list("atmospherics", "bartender", "barber", "blueshield",	"brig_physician", "captain",
+				"cargo", "chief_engineer",	"chaplain",	"chef", "chemist", "civilian", "clown", "cmo", "coroner", "detective", "engineering", "genetics", "hop",
+				"hos", "hydroponics", "internal_affairs_agent", "janitor",	"magistrate", "mechanic", "medical", "mime", "mining", "nt_representative", "paramedic", "pod_pilot",
+				"prisoner",	"research_director", "security", "syndicate", "therapist", "virology", "warden", "xenobiology")
```

Comment:
Actually I've been trying to get the replace text proc to work right on this for over an hour now hehe so far im at  **decalselection = replacetext(decalselection, "\\s", "\\x5F")** but for whatever reason it's not returning the correct result. lowertext works fine though.

Any idea on what I'm doing wrong?